node-runner-cli/config/GatewayDockerConfig.py

```diff
@@ -17,15 +17,15 @@ def ask_postgress_settings(self, postgress_password):
         Helpers.section_headline("POSTGRES SETTINGS")
         if "DETAILED" in SetupMode.instance().mode:
             self.setup, self.host = Prompts.ask_postgress_location(self.host)
-            self.user = Prompts.get_postgress_user()
             self.dbname = Prompts.get_postgress_dbname()
+            self.user = Prompts.get_postgress_user()
         if not postgress_password:
             self.password = Prompts.ask_postgress_password()
         else:
             self.password = postgress_password
 
 
-class CoreApiNode(BaseConfig):
+class CoreApiNodeConfig(BaseConfig):
```

Comment:
Why do we have CoreApiNodeConfig within GatewayDockerConfig? Wouldn't it be better if it lived in it's own python file. Then in here we could use mixins as a for constructor GatewayDockerConfig